x-pack/plugins/security_solution/common/api/asset_criticality/common.schema.yaml

```
@@ -0,0 +1,61 @@
+openapi: 3.0.0
+info:
+  title: Asset Criticality Common Schema
+  description: Common schema for asset criticality
+  version: 1.0.0
+paths: { }
+components:
+  parameters:
+    id_field:
+      name: id_value
+      in: query
+      required: true
+      schema:
+        type: string
+      description: The ID value of the asset.
+    id_value:
+      name: id_field
+      in: query
+      required: true
+      schema:
+        type: string
+      example: 'host.name'
+      description: The field representing the ID.
```

I may be missing something, but I think these need to be swapped?

```suggestion
    id_value:
      name: id_value
      in: query
      required: true
      schema:
        type: string
      description: The ID value of the asset.
    id_field:
      name: id_field
      in: query
      required: true
      schema:
        type: string
      example: 'host.name'
      description: The field representing the ID.
```

---

resolved in https://github.com/elastic/kibana/pull/172073/commits/4a329af236dcc6eaf6444a090f02ade8d3642039